homeassistant/core.py

```diff
@@ -230,11 +231,31 @@ def async_add_job(
 
         return task
 
+    @callback
+    def async_create_task(self, target: Coroutine) -> asyncio.tasks.Task:
+        """Create a task from within the eventloop.
+
+        This method must be run in the event loop.
+
+        target: target to call.
+        """
+        if asyncio.iscoroutine(target):
+            task = self.loop.create_task(target)
+        else:
+            raise ValueError(
+                "async_create_task can be called on coroutine only.")
+
+        # If a task is scheduled
```

Comment:
`# If we're tracking tasks` or one can drop the comment

Reply:
Done